### ISSUE TITLE
Upgrade JHipster Registry code to sync with JHipster v7.2

### ISSUE THREAD

**Issue by vishal423** [jhipster/jhipster-registry#513]
##### **Overview of the issue**

The latest spring boot upgrade includes an update in the oauth2 dependencies, and that's required to upgrade Keycloak on the JHipster side.

##### **Related issues**

https://github.com/jhipster/generator-jhipster/pull/15836
##### **Browsers and Operating System**

- [ ] Checking this box is mandatory (this is just to show you read everything)

**Comment by jdubois** [comment]
Adding a bounty on this

**Comment by pascalgrimaud** [comment]
Cc @juliensadaoui too, as he's the stream lead

**Comment by pascalgrimaud** [comment]
Increasing the bounty as it's more work than expected

**Comment by pascalgrimaud** [comment]
@vishal423 : is this PR https://github.com/jhipster/jhipster-registry/pull/515 enough for making Registry works with Keycloak 15 ? What's the state of this ticket ?

**Comment by vishal423** [comment]
@pascalgrimaud, Yes, with referenced PR the Keycloak upgrade is unblocked. I am a little occupied on other fronts to dedicate time for this ticket. I will check later this week, or if someone has bandwidth feel free to do.

**Comment by pascalgrimaud** [comment]
@vishal423 : so we can close this ticket so the contributor can claim the bounty. And we can ooen a new one, which will be dedicated to upgrade to last JH version. What do you think?

**Comment by vishal423** [comment]
I think we can assign a separate bounty on that PR and keep this ticket as is. This ticket is more about upgrading both frontend and backend code to sync with latest generator release and that's still pending.

**Comment by pascalgrimaud** [comment]
Ok you're right. Let's keep this ticket

**Comment by vishal423** [comment]
I believe this is complete with https://github.com/jhipster/jhipster-registry/pull/518 Can we close this ticket?

**Comment by mshima** [comment]
@pascalgrimaud bounty claimed https://opencollective.com/generator-jhipster/expenses/52433

**Comment by pascalgrimaud** [comment]
@mshima : approved
